railties/CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 *   Make railsrc file location xdg-specification compliant
 
-    Rails new will load the railsrc file from XDG_CONFIG_HOME/rails/railsrc
-    It will then fallback to .config if XDG_CONFIG_HOME not specified
-    It will then fallback to HOME/.railsrc
+    Rails new will load the `railsrc` file from `XDG_CONFIG_HOME/rails/railsrc`.
+    It will then fallback to `.config` if `XDG_CONFIG_HOME` not specified.
+    It will then fallback to `HOME/.railsrc`.
```

Comment:
"Rails new" could use some backticks too!  Also, to clarify what `.config` means, what do you think about:

```suggestion
    `rails new` will now look for the default `railsrc` file at
    `$XDG_CONFIG_HOME/rails/railsrc` (or `~/.config/rails/railsrc` if
    `XDG_CONFIG_HOME` is not set).  If this file does not exist, `rails new`
    will fall back to `~/.railsrc`.
```

Reply:
Sure, I've made these changes now.